### DISCUSSION TITLE
how to bring connection shortcuts to linux ?

### DISCUSSION THREAD

**Discussion by Hanibachi** [rustdesk/rustdesk#7049] [Q&A] [answered]
i need a simpler way to open connection for someone very ignorant in the ways of tech.
i thought i d create an executable shortcut that starts the connection by a terminal command, but i cant find any useful documentation on any available terminal commands for rustdesk .
using latest version on fedora 39.any ideas ?

**Comment by twprh** [comment]
start rustdesk with following parameters:
rustdesk --connect 1234567 --password password
(1234567 is the ID and password the password)

**Reply by Hanibachi** [reply]
there isnt a rustdesk command in bash

**Comment by Hanibachi** [comment, marked as answer]
the answer is : you have to launch the executable with the parameters you want.
using the flatpack version, you can find the executable in : **var/lib/flatpak/exports/bin/com.rustdesk.RustDesk**
- cd into the location :
 `cd var/lib/flatpak/exports/bin/`
- launch rustdesk with parameters you want :
`./com.rustdesk.RustDesk --connect **id** --password **password**`

the other parameters are available here [rustdesk documentation](https://rustdesk.com/docs/en/client/#command-line-parameters)
